.github/workflows/test.yml

```diff
@@ -161,7 +156,8 @@ jobs:
         run: |
           nix --version
           export JQ=$(nix-build '<nixpkgs>' -A jq --no-link)/bin/jq
-          booster=$(nix build .#kore-rpc-booster --extra-experimental-features 'nix-command flakes' --print-build-logs --json | $JQ -r '.[].outputs | to_entries[].value')
+          nix build .#kore-rpc-booster --extra-experimental-features 'nix-command flakes' --print-build-logs
+          booster=$(nix build .#kore-rpc-booster --extra-experimental-features 'nix-command flakes' --json | $JQ -r '.[].outputs | to_entries[].value')
```

Comment:
I don't understand this change. Why is it better to separate the steps of building and extracting the executable path?

Reply:
I believe @jberthold pointed out this can fail but the CI won't pick up on it. This way, it hopefully fails correctly.